erystrea_bot.py

```diff
@@ -29,6 +30,7 @@ async def on_ready():
     print('We have logged in as {0.user}'.format(client))
 
 @client.event
+@commands.cooldown(1, 120, commands.BucketType.user)
```

Comment:
I tested it, and it doesn't work. As currently, the bot doesn't use commands API (reacting for messages with specific prefix), this probably doesn't work. I would suggest adding the code directly to the function, setting some cooldown timer maybe?